sympy/polys/tests/test_rings.py

```diff
@@ -1363,6 +1363,28 @@ def test_PolyElement_pdiv():
     assert f.pquo(g) == q
     assert f.pexquo(g) == q
 
+
+def test_PolyElement_sparse():
```

Comment:
This test function name is not really appropriate. PolyElement is always sparse. What is tested here is `coeff_wrt` and `prem`. Perhaps make separate test functions for each.